go/pserver/optimizer.go

```diff
@@ -1,42 +1,72 @@
 package pserver
 
-/*
-#include "optimizer.h"
-*/
+// #cgo pkg-config: protobuf
+// #cgo CFLAGS: -I ../../
+// //FIXME: ldflags contain "build" path
+// #cgo LDFLAGS: ../../build/go/pserver/cclient/libpaddle_go_optimizer.a -lstdc++
```

Comment:
there is no any part use math library, I cannot reproduce it in my computer and docker environment.  Will talk with you offline
anyway, add the flags.